sdk/eventhub/Azure.Messaging.EventHubs/design/event-processor{T}-proposal.md

```diff
@@ -424,12 +430,11 @@ public class EventProcessorOptions
     public TimeSpan MaximumWaitTime { get; set; } = TimeSpan.FromSeconds(60);
     public EventPosition DefaultStartingPosition { get; set; } = EventPosition.Earliest;
     public TimeSpan LoadBalancingUpdateInterval { get; set; } = TimeSpan.FromSeconds(10);
-    public TimeSpan PartitionOwnershipExpirationInterval { get; set; } = TimeSpan.FromSeconds(30);
```

Comment:
Why are we removing this? Do we intend to let users override this value using some other way?

Reply:
Yeah... it looks like that that's a mistake.